Assets/AnkrSDK/Examples/Scripts/UI/DisconnectButtonController.cs

```diff
@@ -14,49 +16,34 @@ public class DisconnectButtonController : MonoBehaviour
 
 		private void OnEnable()
 		{
-			SubscribeOnTransportEvents().Forget();
+			SubscribeEvents().Forget();
 
 			_button.onClick.AddListener(OnButtonClick);
 		}
 
 		private void OnDisable()
 		{
-			UnsubscribeFromTransportEvents();
+			UnsubscribeEvents().Forget();
 
 			_button.onClick.RemoveAllListeners();
 		}
 
-		private async UniTaskVoid SubscribeOnTransportEvents()
+		private async UniTaskVoid SubscribeEvents()
 		{
-			await UniTask.WaitUntil(() => WalletConnect.Session != null);
-
-			var walletConnectSession = WalletConnect.Session;
-			if (walletConnectSession == null)
-			{
-				return;
-			}
-
-			walletConnectSession.OnTransportConnect += UpdateDisconnectButtonState;
-			walletConnectSession.OnTransportDisconnect += UpdateDisconnectButtonState;
-			walletConnectSession.OnTransportOpen += UpdateDisconnectButtonState;
+			await UniTask.WaitUntil(() => WalletConnect.Status != WalletConnectStatus.Uninitialized);
```

Comment:
Why do we need this await?

Reply:
It's the same as what we had before:
await UniTask.WaitUntil(() => _walletConnect.Session != null);
I just changed it with status because Session property was removed. It guarantees that the session will already exist when subscription happens, but maybe now it is not necessery.